.github/CODEOWNERS

```
@@ -356,9 +356,9 @@ packages/kbn-ecs @elastic/kibana-core @elastic/security-threat-hunting-investiga
 x-pack/packages/security-solution/ecs_data_quality_dashboard @elastic/security-threat-hunting-investigations
 x-pack/plugins/ecs_data_quality_dashboard @elastic/security-threat-hunting-investigations
 packages/kbn-elastic-agent-utils @elastic/obs-ux-logs-team
-x-pack/packages/kbn-elastic-assistant @elastic/security-solution
-x-pack/packages/kbn-elastic-assistant-common @elastic/security-solution
-x-pack/plugins/elastic_assistant @elastic/security-solution
+x-pack/packages/kbn-elastic-assistant @elastic/security-generative-ai
+x-pack/packages/kbn-elastic-assistant-common @elastic/security-generative-ai
+x-pack/plugins/elastic_assistant @elastic/security-generative-ai
```

```suggestion
x-pack/plugins/elastic_assistant @elastic/security-generative-ai
x-pack/plugins/security_solution/public/assistant @elastic/security-generative-ai
```

Maybe we should add one to `x-pack/plugins/security_solution/public/assistant` too?

---

Yes yes, thank you!

---

Resolved in [c2fa5f1](https://github.com/elastic/kibana/pull/175605/commits/c2fa5f1060742cc12f8eb929b9bdd0fd8efafd2b)!